src/Microsoft.DotNet.Compiler.Common/Executable.cs

```diff
@@ -85,6 +85,7 @@ private static void CopyAllDependencies(string outputPath, IEnumerable<LibraryEx
                 libraryExport.RuntimeAssemblies.CopyTo(outputPath);
                 libraryExport.NativeLibraries.CopyTo(outputPath);
                 libraryExport.RuntimeAssets.StructuredCopyTo(outputPath);
+                libraryExport.CompilationAssets.StructuredCopyTo(outputPath);
```

Comment:
Might be resource dlls, pdb, xml. Things produced by dotnet compile. Playing around with this idea.